cmake/macros/macro_deal_ii_pickup_tests.cmake

```diff
@@ -114,6 +114,26 @@ MACRO(DEAL_II_PICKUP_TESTS)
   ENDIF()
 
   #
+  # Check that the diff programs can run and terminate successfully:
+  #
+  FOREACH(_diff_program ${NUMDIFF_EXECUTABLE} ${DIFF_EXECUTABLE})
+    EXECUTE_PROCESS(COMMAND ${_diff_program} "-v"
+      TIMEOUT 4 # seconds
```

Comment:
that's important, thanks! The issue was that `numdiff` build by Homebrew hanged during running dealiii tests